eos-setup-flatpak-extension-dir-extra.service

```diff
@@ -1,6 +1,7 @@
 [Unit]
 Description=Setup flatpak extension directory on extra storage
 ConditionPathExists=/var/endless-extra/flatpak
+ConditionPathExists=!/var/endless-extra/flatpak/extension
```

Comment:
Could we do:
ConditionPathIsMountPoint=/var/endless-extra
ConditionPathExists=/var/endless-extra/flatpak
ConditionPathIsSymbolicLink=!/var/endless-extra/flatpak/extension

Reply:
Updated!